app/models.py

```diff
@@ -1431,6 +1453,17 @@ def validates_data(self, key, data):
 
         return data
 
+    @validates('awarded_at')
+    def validates_awarded_at(self, key, awarded_at):
+        if self.status == "awarded":
+            raise ValidationError('Brief response award cannot be changed as the brief has already been awarded.')
+        if awarded_at:
```

Comment:
It's probably worth changing this to break out of the function early like:
```
if not awarded_at:
    return None
if self.brief.status != "closed":
    ...
```
Saves you a level of indentation at the cost of another line and some people find it more readable:  https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement

I don't care about indentation level personally but if some people find it easier to read then fine.